model/postgres/postgres_server.rb

```diff
@@ -257,9 +258,19 @@ def run_query(query)
   end
 
   def metrics_config
+    ignored_timeseries_patterns = [
+      "pg_stat_user_tables_.*",
+      "pg_statio_user_tables_.*"
+    ]
+    match_pattern = ignored_timeseries_patterns.join("|")
```

Comment:
The variable name is bit confusing, it feels like we are trying to match those pattern, however we are trying to exclude them.

Reply:
makes sense, renamed